bogenliga/src/app/modules/sportjahresplan/components/schusszettel/schusszettel.component.ts

```diff
@@ -141,13 +141,24 @@ export class SchusszettelComponent implements OnInit {
   save() {
     if (this.match1.satzpunkte > 7 || this.match2.satzpunkte > 7) {
       this.notificationService.showNotification({
-        id: 'NOTIFICATION_SCHUSSZETTEL_ENTSCHIEDEN',
-        title: 'SPORTJAHRESPLAN.SCHUSSZETTEL.NOTIFICATION.ENTSCHIEDEN.TITLE',
+        id:          'NOTIFICATION_SCHUSSZETTEL_ENTSCHIEDEN',
+        title:       'SPORTJAHRESPLAN.SCHUSSZETTEL.NOTIFICATION.ENTSCHIEDEN.TITLE',
         description: 'SPORTJAHRESPLAN.SCHUSSZETTEL.NOTIFICATION.ENTSCHIEDEN.DESCRIPTION',
-        severity: NotificationSeverity.ERROR,
-        origin: NotificationOrigin.SYSTEM,
-        type: NotificationType.OK,
-        userAction: NotificationUserAction.ACCEPTED
+        severity:    NotificationSeverity.ERROR,
+        origin:      NotificationOrigin.SYSTEM,
+        type:        NotificationType.OK,
+        userAction:  NotificationUserAction.ACCEPTED
+      });
+    } else if(this.match1.schuetzen[0][0].schuetzeNr == null || this.match1.schuetzen[1][0].schuetzeNr == null || this.match1.schuetzen[2][0].schuetzeNr == null
+    || this.match2.schuetzen[0][0].schuetzeNr == null || this.match2.schuetzen[1][0].schuetzeNr == null || this.match2.schuetzen[2][0].schuetzeNr == null) {
+      this.notificationService.showNotification({
+        id:          'NOTIFICATION_SCHUSSZETTEL_ENTSCHIEDEN',
+        title:       'SPORTJAHRESPLAN.SCHUSSZETTEL.NOTIFICATION.ENTSCHIEDEN.TITLE',
+        description: 'Bitte unter Schuetze in alle Felder eine Schuetzennummer eintragen!', //hier ist die Meldung direkt eingefügt
```

Comment:
Bitte die Meldung auch in die de.json eintragen per Key (so wie oben die description)
Datei liegt unter src/assets/i18n/de.json

Reply:
Alles klar